source/includes/examples/_overview.html.md

```diff
@@ -2,6 +2,8 @@
 
 This page contains working examples of Inkling code in conjunction with python simulator files. All of these examples and the libraries that accompany them can be found on [BonsaiAI's GitHub page][1] and are also linked within each Example.
 
+All of the **OpenAI Gym Examples** and **EnergyPlus** can be trained in the cloud with Bonsai Managed Simulators. A full list of supported Docker containers for Managed Simulators can be found on [Quay][3].
```

Comment:
I'm not sure that linking to the bonsai organization page on Quay here is going to clearly explain to users the 'full list of supported docker containers'. We use quay for hosting images other then simulator images: we've got 9 public images, but only three of them are simulator containers.

It might be better to link to the list that you've added in _projectfile.html.md